config.production.php

```diff
@@ -3,7 +3,7 @@
 return [
     'production' => true,
     'matomo_container' => '8jNjdh8C',
-    'accountUrl' => 'https://account.libresign.coop',
+    'accountUrl' => getenv('ACCOUNT_URL'),
```

Comment:
Production should be hardcode. If not, we will need to add this environment at GitHub Actions.
```suggestion
```